scripts/exrrfs_ic.sh

```diff
@@ -3,10 +3,10 @@ declare -rx PS4='+ $(basename ${BASH_SOURCE[0]:-${FUNCNAME[0]:-"Unknown"}})[${LI
 set -x
 cpreq=${cpreq:-cpreq}
 if [[ -z "${ENS_INDEX}" ]]; then
-  prefixin=${IC_PREFIX:-IC_PREFIX_not_defined}
+  prefixin=${SOURCE:-IC_PREFIX_not_defined}
```

Comment:
I think `IC_PREFIX` is more clear than `SOURCE`

Reply:
Anyway, `source` itself will bring confusions. `EXT_SOURCE` or similar may work a little bit better.

Reply:
Change to "EXTRN_MDL_SOURCE". Hope this one make more sense.